### ISSUE TITLE
[BUG] Unable to load library 'libSkiaSharp'.

### ISSUE THREAD

**Issue by jakobeazy** [mono/SkiaSharp#2758]
### Description

using svg.skia on anASP.NET website i get the following error:
Exception message: The type initializer for 'SkiaSharp.SKImageInfo' threw an exception.
Unable to load library 'libSkiaSharp'.

The strange thing is that debugging the code locally, it works, but when deploying it to a server it fails with this error.
The versions i an using:
Svg.Skia: 1.0.0.13
SkiaSharp: 2.88.6
SkiaSharp.HarfBuzz: 2.88.6
SkiaSharp.NativeAssets.macOS: 2.88.6
SkiaSharp.NativeAssets.Win32: 2.88.6

the project is a .NET Framework 4.7.2
I can't figure out why it can run locally and not remote.

### Code

The best way to share code for larger projects is a link to a GitHub repository: https://github.com/user/repo/tree/bug-123

But, you can also share a short block of code here:
```cs
// some C# code here
```
You can also share some XAML:
```xaml
<!-- xaml code here -->
```

### Version of SkiaSharp

Other (Please indicate in the description)

### Last Known Good Version of SkiaSharp

Other (Please indicate in the description)

### IDE / Editor

Visual Studio (Windows)

### Platform / Operating System

Windows

### Platform / Operating System Version

Windows Server 2016 v1607

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mattleibow** [comment]
Duplicate of https://github.com/mono/SkiaSharp/issues/2385

**Comment by mattleibow** [comment]
Is the libSkiaSharp.dll file in the bin folder somewhere? How are you publishing/deploying? Are you able to share a msbuild binlog so I can see why the native binary is not reaching the final output.

If you run the app locally after publishing, does it work?

**Comment by romansp** [comment]
@mattleibow We're also seeing this error but when trying out `3.0 preview 2`. 

It's a `browser-wasm` app on .Net 8. Loads fine on `2.88.7` and HarfBuzzSharp `7.3.0.1`. We don't use Blazor but we do include reference to `SkiaSharp.Views.Blazor` for `InterceptGLObject` workaround and that's where it fails to load `libSkiaSharp`.

Haven't tried publishing yet as we're seeing this error with regular `dotnet build`.

Didn't want to bother early because it's still a preview. But can create a minimal repro if that's already expected to work properly.

**Comment by jakobeazy** [comment]
hi
I was actually using the svg.skia nuget package. Changing to use the svg package solved it for me.

**Comment by prashants7718** [comment]
Hi @jakobeazy ,

How you have solved this issue, have you changed package and used another package for svg generation

**Comment by jakobeazy** [comment]
Yes, i changed the nuget package from svg.skia to svg.

**Comment by mattleibow** [comment]
@romansp it should be working since there should be no changes for wasm in 3.x Please attach a repro/sample so I can have a look.

**Comment by enghch** [comment]
@mattleibow I have the same issue when trying to run a Wasi Console app that depends on SkiaSharp. It builds the single-file wasm just fine. But when attempting to run, it throws:

`"System.TypeInitializationException: TypeInitialization_Type, SkiaSharp.SKImageInfo --- System.DllNotFoundException: libSkiaSharp   at SkiaSharp.SKImageInfo..cctor()`


I'm on .NET 9 preview 6. Is there some workaround or something I'm missing? I attached a small repro project. I included the wasm file in case that's useful. If you're not familiar with wasi or wasitime, see in the instructions in the readme.

[WasiSkiaSharpRepro.zip](https://github.com/user-attachments/files/17076962/WasiSkiaSharpRepro.zip)
